config/routes.rb

```diff
@@ -2,5 +2,8 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] Here it is a requirement that we do not expose unnecessary endpoints. Kindly only include the `index` and `show` endpoints for both the `Post` and the `Users`

e.g
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```